src/utils/common_utils.h

```diff
@@ -0,0 +1,9 @@
+#ifndef UTILS_COMMON_UTILS_H
+#define UTILS_COMMON_UTILS_H
+
+typedef struct{
+	int x;
+	int y;
+}point_t;
```

Comment:
Maybe this struct should be moved to `src/utils/geometry`. This doesn't make a difference right now, but should come in handy once path finding is implemented.

Path finding has (could, and in my opinion, should have) abstractions beyond adding a point, like adding a hexagon, or a rectangle.